graylog2-web-interface/src/components/cluster/index.js

```diff
@@ -0,0 +1 @@
+export GraylogClusterOverview from './GraylogClusterOverview';
```

Comment:
Babel 6 is the one that matters the most for us. I am remembering we talked about this briefly when I changed most of the exports to the format.